multinet/multinet/__init__.py

```diff
@@ -15,14 +15,32 @@
 from .schema import schema
 from . import db
 
+
+def graphql_query(query):
+    result = graphql(schema, query)
+    if result:
+        errors= [error.message for error in result.errors] if result.errors else []
+        logprint("Errors in request: %s" % len(errors), level=logging.WARNING)
+        for error in errors[:10]:
+            logprint(error, level=logging.WARNING)
+    else:
+        errors = []
+    return dict(data=result.data, errors=errors)
+
+
 class MultiNet(Resource):
     def __init__(self, port):
         super(MultiNet, self).__init__()
         self.resourceName = 'multinet'
         self.arango_port = port
         self.route('POST', ('graphql',), self.graphql)
         self.route('POST', ('bulk', ':workspace', ':table'), self.bulk)
-        self.route('POST', ('tree', ':workspace', ':table'), self.tree)
+
+        # Newick tree operations.
+        self.route('POST', ('newick', 'tree', ':workspace', ':table'), self.tree)
+
+        # Operations for Juniper application.
+        self.route('POST', ('juniper', 'node'), self.juniper_get_node)
```

Comment:
As discussed in chat, we would like to drop this endpoint in favor of moving this logic to the client and possibly expanding into new graphql query types.

Reply:
Reverted the commit.